### ISSUE TITLE
undo-ing after creating interaction or mark crashes

### ISSUE THREAD

**Issue by jonathanzong** [uwdata/lyra#502]
Steps to repro:

- create a plot and click Add Interaction (don't demonstrate anything)
- click undo until it crashes

error: 

```
Uncaught TypeError: interaction is undefined
    mapStateToProps Interaction.tsx:19
    Redux 6
    React 38
    tsx index.tsx:14
    Webpack 11
Interaction.tsx:19
    mapStateToProps Interaction.tsx:19
    Redux 6
    React 13
    dispatchInteractiveEvent self-hosted:1208
    React 25
    tsx index.tsx:14
    Webpack 11
```

**Comment by jonathanzong** [comment]
this is a general problem where the currently open inspector has a reference to an ID, and when undo removes something that was created, that ID will no longer point at a record in the store (causing reference errors down the line)

**Comment by jonathanzong** [comment]
fixed in 77e051f85ad86d460782551bd98ceb3f43cff7c0
